src/core/services/interaction/interaction.js

```diff
@@ -0,0 +1,58 @@
+angular
+  .module('material.core.interaction', [])
+  .service('$mdInteraction', MdInteractionService);
+
+function MdInteractionService($timeout) {
+  var body = angular.element(document.body);
+  var _mouseEvent = window.MSPointerEvent ? 'MSPointerDown' : window.PointerEvent ? 'pointerdown' : 'mousedown';
+  var buffer = false;
+  var timer;
+  var lastInteractionType;
+  var inputMap = {
+    'keydown': 'keyboard',
+    'mousedown': 'mouse',
+    'mouseenter': 'mouse',
+    'touchstart': 'touch',
+    'pointerdown': 'pointer',
+    'MSPointerDown': 'pointer'
+  };
+  var pointerMap = {
+    2: 'touch',
```

Comment:
@EladBezalel Can I contact you on something else, I mean (live) thats too slow :smile: